addons/respawn/stringtable.xml

```diff
@@ -334,8 +334,21 @@
             <Chinese>擺放此模塊後，當有發生友軍誤擊致死的事件，會顯示提示訊息在聊天視窗中</Chinese>
         </Key>
         <Key ID="STR_ACE_Respawn_FriendlyFire_Message">
-            <English>%1 was killed by %2</English>
-            <German>%1 wurde durch %2 getötet</German>
+          <Original>%1 was killed by %2</Original>
```

Comment:
We don't use Original. We just put English as first entry, so that is chosen as default automatically by virtue of Original being undefined.